### DISCUSSION TITLE
Adding milk to barrels

### DISCUSSION THREAD

**Discussion by remplerus** [NovaMachina-Mods/ExNihiloSequentia#350] [Ideas]
This is a idea thread for adding milk to barrels. As there's a open issue, I moved it to this discussion thread. (Linked #347 )

**Comment by TimeheroTH** [comment]
Sorry but milk could already bee added to barrels with a minecraft bucket, the problem is that only with ceramic/wooden buckets doesn't work @cech12 @remplerus

**Reply by cech12** [reply]
@TimeheroTH I dont know, which mods you are using, but milk cannot be added into barrels by Ex Nihilo. (see the other answer)

**Comment by remplerus** [comment]
milk should be inserted into barrels with any bucket mod. if not, it's not our fault. therefore i lock this idea as completed.
Side note: in waila/top, milk doesn't show up as milk, but as air. i made a PR to change that. #352
